archive-editorial.php

```diff
@@ -4,16 +4,16 @@
 
 <!-- main content -->
 
-<main id="main-content">
+<main id="main-content" class="container">
```

Comment:
in some templates `.container` is on `<main>` and others on `<section>` whats the logic?